dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/preheat/hooks/FileResourceTrackerPreheatHook.java

```diff
@@ -72,14 +72,17 @@ public void preheat( TrackerPreheatParams params, TrackerPreheat preheat )
 
         List<String> fileResourceIds = new ArrayList<>();
 
-        params.getTrackedEntities().forEach( te -> collectResourceIds( fileResourceAttributes, fileResourceIds, te.getAttributes() ) );
-        params.getEnrollments().forEach( en -> collectResourceIds( fileResourceAttributes, fileResourceIds, en.getAttributes() ) );
+        params.getTrackedEntities()
+            .forEach( te -> collectResourceIds( fileResourceAttributes, fileResourceIds, te.getAttributes() ) );
+        params.getEnrollments()
```

Comment:
Technically DataElement can also be files, but I don't remember if we support this for Tracker specifically yet. @Bekkalizer ?